Makefile

```diff
@@ -17,12 +17,18 @@ dep-up:
 	glide up -v
 
 test:
+	GOOS=windows go build -o fn
+	GOOS=linux GOARCH=amd64 go build -o fn
+	GOOS=darwin go build -o fn
+	GOOS=linux go build -o fn
+	go build -o fn
 	./test.sh
 
 release:
 	GOOS=linux go build -o fn_linux
 	GOOS=darwin go build -o fn_mac
 	GOOS=windows go build -o fn.exe
+	GOOS=linux GOARCH=amd64 go build -o fn_linux_arm64
```

Comment:
Isn't this just the normal arch?  Did you mean to use `arm`?

Reply:
Yeah. Fixing.